charts/guac/values.yaml

```diff
@@ -190,10 +200,26 @@ guac:
     ingest: false
     jobName: ingest-guac-data
 
+  ingress:
+    enabled: false
+    # ingressClassName:
+    # webuiHostname:
+    # apiHostname:
+
+  backend:
+    ent:
+      db-driver: postgres
+      db-address: postgres://guac:guac@host:port/dbName?sslmode=disable
+      db-migrate: true
+      db-debug: false
+
+  pubSubAddr: awssqs://sqs.[aws_region].amazonaws.com/[account_id]/[queue_name]?region=[aws_region]
```

Comment:
similarly here, we would want to default to NATS unless otherwise specified. So that it does not break the current implementation. If the user decided `awssqs` or something else, it should not deploy NATS.

Reply:
defaults to nats or take override